lib/builderator/config/defaults.rb

```diff
@@ -74,6 +74,11 @@ module Config
 
             build.ssh_username 'ubuntu'
 
+            # Packer default is 300 seconds.  Specify as a string to give units
+            # such as s (seconds), ms (milliseconds), ns (nanoseconds), etc.
+            # Ints will be interpreted as ns.  Buyer beware.
+            build.ssh_timeout '300s'
```

Comment:
I set it to give an example of the formatting, ie as a string with units.  Is that unreasonable?  Given that we have seen the timeout get hit, I could just change it to 600s.